taxi/urls.py

```diff
@@ -1,9 +1,27 @@
 from django.urls import path
 
-from .views import index
+from .views import (index,
+                    ManufacturerListView,
+                    CarListView, CarDetailView,
+                    DriversListView, DriversDetailView)
```

Comment:
```suggestion
from .views import (
                    index,
                    ManufacturerListView,
                    CarListView, 
                    CarDetailView,
                    DriversListView, 
                    DriversDetailView
                    )
```

Fix indentation like this, it has better readability